iis-2vm-sql-1vm/azuredeploy.json

```diff
@@ -0,0 +1,635 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "envPrefixName": {
```

Comment:
I see this being used for storage account names. A word of warning since storage account names are the top failure cause due to it's strict rules with globally unique names and 24-character limit. 

If you need to limit this then use the minLength and maxLength properties on the parameter.